rllib/algorithms/appo/appo.py

```diff
@@ -158,15 +175,52 @@ def training(
             self.lambda_ = lambda_
         if clip_param is not NotProvided:
             self.clip_param = clip_param
+            self._learner_hps.clip_param = clip_param
```

Comment:
As discussed offline: Can we use the existing design pattern of having these "compile" methods: `AlgorithmConfig.get_xyz_config`?

For example, we already have a `get_evaluation_config_object` that "compiles" a new config object from `self` + the eval_config overrides and returns it. The same way, we could compile the hp config as needed.

Reply:
I like this approach. We should do that for PPO too.

Reply:
creating an open issue. lets do it.